database/controller.js

```diff
@@ -0,0 +1,26 @@
+const db = require('./index');
+
+module.exports.getImage = (id) => {
+  const queryString = 'SELECT imageUrl FROM images WHERE id = (?)';
+  return db.query(queryString, [id]);
+};
+
+module.exports.addImage = (data) => {
+  // const queryString = 'SELECT imageUrl FROM images WHERE id = (?)';
+  // return db.query(queryString, [id]);
+};
+
+module.exports.updateImage = (id) => {
+  // const queryString = 'SELECT imageUrl FROM images WHERE id = (?)';
+  // return db.query(queryString, [id]);
+};
+
+module.exports.deleteImage = (id) => {
+  // const queryString = 'SELECT imageUrl FROM images WHERE id = (?)';
+  // return db.query(queryString, [id]);
+};
+
+module.exports.getAllImages = (houseID) => {
+  const queryString = 'SELECT imageUrl FROM images WHERE houseID = (?)';
+  return db.query(queryString, [houseID]);
+};
```

Comment:
Wouldn't this be 'SELECT * ...' ?

Reply:
Later you call this with an additional parameter (a callback function).

Reply:
This is for the houses endpoint. I don't think we need any information other than imageUrls, so `SELECT imageUrl` replaces `SELECT *`

Reply:
Fixed this function call in server routes